### PR TITLE
Remove utils.move_file function as it is no longer used in the toolbox

### LINKED ISSUE #336: Replace utils.move_file() with shutil.move()
## Summary

I think that the function [`utils.move_file()`](https://github.com/physiopy/phys2bids/blob/da3a5306a91a046f3956f1d033b6f86b5c14611b/phys2bids/utils.py#L152) could be replaced with `shutil.move()`, which automatically overwrites existing files, as desired in `utils.move_file()`.

## Next Steps

    * Remove the function and associated tests
    * Remove the item from the API docs page
    * Replace the function call throughout the package

### PR DESCRIPTION
**Pull request by viacovella**
#Closes #336 and #closes #337 

## Proposed Changes

  - Removes the in-house built function to move the files from the Utilities collection;
  - Removes the test for in-house built function to move the files from the Unit test for Utilities collection.

### CODE REVIEW

**Review by smoia — APPROVED** [review]
LGTM!

@viacovella can you change the first line in the first PR message to "Closes #336 and closes #337"?

File: phys2bids/utils.py [hunk 1]
```diff
@@ -148,36 +148,6 @@ def check_file_exists(filename):
     if not os.path.isfile(filename) and filename is not None:
         raise FileNotFoundError(f'The file {filename} does not exist!')
```
Comment (smoia) [inline phys2bids/utils.py:150]: Can you add a newline after this one? The style we're adopting asks for two lines between every function.

**Comment by smoia** [comment]
Oh, I spoke too soon :sweat_smile: 

The style test is failing because there was a line missing between two functions. Can you correct that please?

**Comment by eurunuela** [comment]
Thank you @viacovella ! I think this is a very simple refactoring PR and two reviews are not really necessary for this one. What do you think @smoia ?

File: phys2bids/utils.py [hunk 2]
```diff
@@ -147,36 +147,7 @@ def check_file_exists(filename):
...
-        Moves file to new destination
-    """
-    check_file_exists(oldpath + ext)
-
-    if os.path.isfile(newpath + ext):
-        os.remove(newpath + ext)
-
-    os.rename(oldpath + ext, newpath + ext)
-
+
```
Comment (smoia) [inline phys2bids/utils.py:150]: Is there a whitespace here?

**Review by smoia — CHANGES_REQUESTED** [review]
Ok, we're close @viacovella, we're close.

**Comment by smoia** [comment]
@viacovella flake8 is not your friend today!

Check [here](https://app.circleci.com/pipelines/github/physiopy/phys2bids/116/workflows/fa776df9-8275-4f4b-88aa-9c0f505e81f1/jobs/745) to know what the problem is.

In this case, apparently there's a whitespace in the line you added.

Can I ask you what are you using to write your scripts? Sometimes adding some linter plugins helps dramatically in avoiding these little issues. I can help you wet them up.

**Comment by eurunuela** [comment]
> Can I ask you what are you using to write your scripts? Sometimes adding some linter plugin helps dramatically in avoiding these little issues. I can help you wet them up.

I'd also suggest that you tune your IDE settings to show you whitespace. It's incredibly useful for these kinds of errors.

File: phys2bids/utils.py (hunk 2 above)
Comment (viacovella) [reply phys2bids/utils.py]: Yes, there were whitespaces. I deleted them using another editor.

**Review by eurunuela — APPROVED** [review]
The style check passed, so this should be ready.

Thank you @viacovella !

**Comment by smoia** [comment]
Ok, I think this is ready!

Since this is not only your first PR, but your first Brainhack PR, here's a gong for you!

:bellhop_bell: 

(Or the closest thing we can get with GH  emojis)

**Review by smoia — APPROVED** [review]

**Comment by eurunuela** [comment]
This PR won't trigger `auto` right?

**Comment by smoia** [comment]
I have no idea since it's the first `refactoring` we merge. I don't think so though, no.

**Comment by smoia** [comment]
:rocket: PR was released in [`2.3.0`](https://github.com/physiopy/phys2bids/releases/tag/2.3.0) :rocket:

**Pull request merged**